samples/IstioSPIRE.yaml

```diff
@@ -1,3 +1,5 @@
+#Pre-requisite: A Kubernetes cluster 
+#Description: This Patternfile installs Spire with the default configuration
```

Comment:
```suggestion
#Description: This design deploys a SPIRE-configured Istio service mesh with the default SPIRE configuration.
```

Reply:
Actually it does not deploy Istio service mesh. It just deploys Spire server and agent configured and ready for an Istio deployment.

Reply:
Oh, I see.